gliderpy/plotting.py

```diff
@@ -90,3 +90,43 @@ def plot_transect(
     ax.set_ylim(ax.get_ylim()[0], 0)
 
     return fig, ax
+
+
+@register_dataframe_method
+def plot_ctd(
+    df: pd.DataFrame,
+    idx: int,
+    var: str,
+    ax: plt.Axes = None,
+    color: str = None
+) -> tuple:
+    """Make a CTD profile plot of pressure vs property
+    depending on what variable was chosen.
+    :param idx: index of position  
```

Comment:
Describe that `idx` is a profile number.

```suggestion
    :param idx: index of position  
```